schemas/styles/csl-choose.rnc

```diff
@@ -0,0 +1,84 @@
+namespace a = "http://relaxng.org/ns/compatibility/annotations/1.0"
+namespace cs = "http://purl.org/net/xbiblio/csl"
+
+
+## cs:choose - Conditional Statements"
+div {
+  rendering-element.choose =
+    
+    ## Use to conditionally render rendering elements.
+    element cs:choose { choose.if, choose.else-if*, choose.else? }
+  
+  ## When specifying conditions, we allow either the attribute syntax or the element-based one.
+  choose.condition = condition.atts | condition.elem
```

Comment:
Does this permit _both_ attribute syntax or element syntax?

Reply:
My intention is only allows one or the other. That's what we want; right?

But to repeat, I haven't tested it. Working on it.

Reply:
Yes, one or the other.

Reply:
Yes, one or the other. Just want to be sure that's enforced.